cassovary-core/src/main/scala/com/twitter/cassovary/graph/ArrayBasedDirectedGraph.scala

```diff
@@ -51,9 +51,12 @@ private case class NodesMaxIds(nodesInOnePart: Seq[Node],
 
 object ArrayBasedDirectedGraph {
   def apply(iteratorSeq: Seq[() => Iterator[NodeIdEdgesMaxId]], parallelismLimit: Int,
-            storedGraphDir: StoredGraphDir):
+            storedGraphDir: StoredGraphDir, neighborsSorted: Boolean = false):
   ArrayBasedDirectedGraph = {
-    val constructor = new ArrayBasedDirectedGraphConstructor(iteratorSeq, parallelismLimit, storedGraphDir)
+    val constructor = new ArrayBasedDirectedGraphConstructor(iteratorSeq,
```

Comment:
do you remember why we have this extra private class whose apply() is called to produce ArrayBasedDirectedGraph (as opposed to just having the code in the constructor inline here)